src/app/pages/event/ingest/index.vue

```diff
@@ -0,0 +1,179 @@
+<template>
+  <div class="space-y-6">
+    <div class="space-y-4">
+      <div v-if="!selectedFile" class="space-y-4">
+        <h2 class="text-lg font-semibold">
+          {{ t('addImages') }}
+        </h2>
+        <div
+          class="cursor-pointer rounded-xl border-2 border-dashed border-gray-300 p-12"
+          @click="triggerFileInput"
+        >
+          <div class="flex flex-col items-center justify-center text-center">
+            <div class="mb-4">
+              <svg
+                class="h-12 w-12 text-gray-400"
+                viewBox="0 0 24 24"
+                fill="none"
+                stroke="currentColor"
+              >
+                <rect x="3" y="3" width="18" height="18" rx="2" ry="2" />
+                <circle cx="8.5" cy="8.5" r="1.5" />
+                <path d="M20.4 14.5L16 10 4 20" />
+              </svg>
+            </div>
+            <div class="space-y-2">
+              <h3 class="text-xl font-medium">{{ t('uploadFiles') }}</h3>
+              <p class="text-sm text-gray-500">{{ t('fileTypes') }}</p>
+            </div>
+          </div>
+        </div>
+        <div class="flex w-full">
+          <ShadButton
+            variant="secondary"
+            class="ml-auto cursor-pointer"
+            @click="triggerFileInput"
+          >
+            {{ t('selectFromDevice') }}
+          </ShadButton>
+        </div>
+      </div>
+
+      <div v-else>
+        <div class="w-full">
+          <div
+            class="relative w-full overflow-hidden rounded-md border border-[#27114F]"
+          >
+            <img :src="previewUrl" class="w-full object-cover" />
+          </div>
+        </div>
```

Comment:
It doesn't need to be complex here:
```suggestion
      <img class="rounded-md" :src="previewUrl" />
```